components/infobox/wikis/starcraft/infobox_map_custom.lua

```diff
@@ -109,10 +109,19 @@ end
 ---@param args table
 ---@return string[]
 function CustomMap:getWikiCategories(args)
+	local categories = {}
+
 	local players = args.players or self:_tlpdMap(args.id, 'players')
-	if String.isEmpty(players) then return {} end
+	if not String.isEmpty(players) then
+		table.insert(categories, 'Maps (' .. players .. ' Players)')
+	end
+
+	local tileset = args.tileset or self:_tlpdMap(args.id, 'tileset')
+	if not String.isEmpty(tileset) then
+		table.insert(categories, tileset .. ' Tileset')
+	end
 
-	return {'Maps (' .. players .. ' Players)'}
+	return categories
 end
```

Comment:
```lua
---@param args table
---@return string[]
function CustomMap:getWikiCategories(args)
	local players = args.players or self:_tlpdMap(args.id, 'players')
	local tileset = args.tileset or self:_tlpdMap(args.id, 'tileset')

	return Array.append({}
		String.isNotEmpty(players) and ('Maps (' .. players .. ' Players)') or nil,
		String.isNotEmpty(tileset) and (tileset .. ' Tileset') or nil
	)
end
```

imo slightly better code style

Reply:
Is there a way for me to accept your changes?
Do I have to make a new PR or commit?
I'm a PR noob 😳

Reply:
commit into the existing pr
at https://github.com/Liquipedia/Lua-Modules/pull/4443/files you see the changes and there you can also click to edit the file:
![IMG_5023](https://github.com/user-attachments/assets/9a36044d-9558-4cfb-a70d-1bbb3faf6761)

Reply:
if you use vscode (or such) just switch to the branch used for this pr
do the edit and commit and sync/push

Reply:
Thank you!
I did it on GitHub